package.json

```diff
@@ -12,15 +12,12 @@
     "jupyterlab-extension"
   ],
   "scripts": {
-    "build": "jlpm run build:lib",
-    "build:labextension": "cd jupyterlab_git && rimraf labextension && mkdirp labextension && cd labextension && npm pack ../..",
-    "build:lib": "tsc",
-    "build:all": "jlpm run build:labextension",
-    "clean": "jlpm run clean:lib",
-    "clean:lib": "rimraf lib && rimraf tsconfig.tsbuildinfo",
+    "build": "jlpm install && tsc",
+    "build:labextension": "jlpm build && jlpm clean:labextension && mkdirp jupyterlab_git/labextension && cd jupyterlab_git/labextension && npm pack ../..",
+    "clean": "rimraf lib && rimraf tsconfig.tsbuildinfo",
+    "clean:more": "jlpm clean && rimraf build && rimraf dist",
     "clean:labextension": "rimraf jupyterlab_git/labextension",
-    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
-    "prepare": "jlpm run clean && jlpm run build",
+    "clean:slate": "jlpm clean:more && jlpm clean:labextension && rimraf node_modules",
```

Comment:
More for my information, why `slate` ?

Reply:
It's in analogy to the `clean:slate` script in Jlab core; this is the command that restores the repo as a whole to a "pre-build" state. `rimraf node_modules` has to go last, since after you get rid of `node_modules` you lose the `rimraf` command itself.

As for the name "slate" itself, I didn't come up with it, but I assume that it comes from the Latin idiom *tabula rasa*. This is usually rendered in English as "blank slate" or "clean slate". People often say something like "Starting from a clean slate..." to mean starting (over) from an initial state.

Reply:
Thx for this detailed answer 😃